benches/bench.rs

```diff
@@ -63,19 +65,27 @@ fn v2_serialize(req: RequestSer<'_>) -> String {
 }
 
 pub fn jsonrpsee_types_v2(crit: &mut Criterion) {
-	crit.bench_function("jsonrpsee_types_v2_array_ref", |b| {
+	// Construct the serialized request using the `ParamsSer` directly.
+	crit.bench_function("jsonrpsee_types_baseline_params", |b| {
 		b.iter(|| {
 			let params = &[1_u64.into(), 2_u32.into()];
 			let params = ParamsSer::ArrayRef(params);
+			let params = serde_json::to_string(&params).unwrap();
+			let params = serde_json::value::RawValue::from_string(params).unwrap();
```

Comment:
so here I thought we could just manually write eg "[1,2]" and convert that to a RawValue and then we'd have a baseline doing the "fastest thing" to benchmark our ArrayParamsBuilder against :)

we could do the same for ObjectParams too but we can add that another time!